static/src/js/components/GranuleDetails/GranuleDetailsMetadata.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './GranuleDetailsMetadata.scss'
+
+export const GranuleDetailsMetadata = ({ metadataUrls }) => {
+  const metdataUrlKeys = [
+    'native',
+    'umm_json',
+    'atom',
+    'echo10',
+    'iso19115'
+  ]
+  return (
+    <div className="granule-details-metadata">
+      <div className="granule-details-metadata__content">
+        <h4 className="granule-details-metadata__heading">Download Metadata:</h4>
+        <ul className="granule-details-metadata__list">
+          {
+            metdataUrlKeys.length && metdataUrlKeys.map((key) => {
+              const metadataUrl = metadataUrls[key]
+              return (
+                <li
+                  key={`metadata_url_${metadataUrl.title}`}
+                  className="granule-details-metadata__list"
+                >
+                  <a href={metadataUrl.href}>
```

Comment:
This `metadataUrl` may require a token to be accessed. While authentication isn't available yet we need to ensure that it gets documented (likely a new ticket @mreese84 ) to come back and add ensure that it's feasible. This also applies to the single granule downloading.

Reply:
I left a TODO in `static/src/js/util/granules.js` as a reminder and I have added a note in our TODO spreadsheet as well.